src/ui/Windows/ThisMfcApp.cpp

```diff
@@ -1273,7 +1273,7 @@ void ThisMfcApp::ClearMRU()
   }
 
   // Recent databases are on the popup menu off the main File menu
-  CMenu *pPopupMenu = pFile_Submenu->GetSubMenu(3);
+  CMenu *pPopupMenu = pFile_Submenu->GetSubMenu(FindMenuItem(pFile_Submenu, ID_MENUITEM_LOCK) + 2);
```

Comment:
1. I wish there was a more elegant/robust solution to finding the menuItem instead of assuming it's after the Lock menu item, but I can't think of one
2. Where does "+ 2" come from? It works, but I don't understand why "+ 1" doesn't work?
3. If FindMenuItem fails to find an item, it will return -1. This will mean GetSubMenu() will be called with -1 + 2 = 1, which will not be correct. Perhaps an ASSERT on FindMenuItem() would be better protection against future menu changes?

Reply:
I think a slightly safer assumption is that the menu immediately precedes the "Clear Recently Opened List" entry.